trpcultivate_phenotypes/tests/src/Kernel/Validators/Traits/ValidatorTraitFileTypesTest.php

```diff
@@ -307,13 +324,20 @@ public function provideMimeTypeForSetter() {
 
     // For each senario we expect the following:
     // -- scenario label to provide helpful feedback if a test fails.
-    // -- a string that is the mime-type to pass to setFileMimeType() and that
-    //    we also expect to have returned by getFileMimeType()
-    // -- an array indicating whether to expect an exception with the keys
-    //    being the method and the value being TRUE if we expect an exception
-    //    when calling it for this senario.
-    // -- an array of expected exception messages with the key being the method
-    //    and the value being the message we expect (NULL if no exception expected)
+    // -- a string that is the mime-type to pass to setFileMimeType()
```

Comment:
This is my first time to see this notation (double dash and single dash lines). Is this used for describing elements in data providers?

Reply:
I just had a quick chat with Lacey about this and it isn't a standard for describing data providers, it's just what was intuitive at the time. Your way of using just single dashes in the Headers trait tests is also perfectly fine! 😄